utils/ioutils/ioutils.go

```diff
@@ -41,7 +42,7 @@ func ScanJFrogPasswordFromConsole() (string, error) {
 }
 
 func ScanPasswordFromConsole(message string) (string, error) {
-	log.Output(message)
+	fmt.Print(message)
```

Comment:
What would you suggest we use instead @sverdlov93 ? We're not supposed to have a new line at the end of every config prompt. It doesn't look good.

Reply:
@eyalbe4 we can change the log.Output() to use print instead of println...
Or maybe pass an optional variable to determine if a new line should be printed?

Reply:
OK. Will you be able to implement in a follow-up PR @talarian1?

Reply:
Sure 👍🏻